python.d/springboot.chart.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+# Description: tomcat netdata python.d module
+# Author: Wing924
+
+import json
+from bases.FrameworkServices.UrlService import UrlService
+
+# default module values (can be overridden per job in `config`)
+# update_every = 2
+priority = 60000
+retries = 60
+
+
+DEFAULT_ORDER = ['response_code', 'threads', 'gc_time', 'gc_ope', 'heap']
+
+DEFAULT_CHARTS = {
+    'response_code': {
+        'options': [None, "Response Codes", "requests/s", "response", "springboot.response_code", "stacked"],
+        'lines': [
+            ["resp.other",  'Other', 'incremental'],
+            ["resp.1xx",    '1xx',   'incremental'],
+            ["resp.2xx",    '2xx',   'incremental'],
+            ["resp.3xx",    '3xx',   'incremental'],
+            ["resp.4xx",    '4xx',   'incremental'],
+            ["resp.5xx",    '5xx',   'incremental'],
+        ]},
+    'threads': {
+        'options': [None, "Threads", "current threads", "threads", "springboot.threads", "area"],
+        'lines': [
+            ["threads.daemon", 'daemon', 'absolute'],
+            ["threads", 'total', 'absolute'],
+        ]},
+    'gc_time': {
+        'options': [None, "GC Time", "milliseconds", "garbage collection", "springboot.gc_time", "area"],
+        'lines': [
+            ["gc.copy.time",                'Copy',                 'incremental'],
+            ["gc.marksweepcompact.time",    'MarkSweepCompact',     'incremental'],
+            ["gc.parnew.time",              'ParNew',               'incremental'],
+            ["gc.concurrentmarksweep.time", 'ConcurrentMarkSweep',  'incremental'],
+            ["gc.ps_scavenge.time",         'PS Scavenge',          'incremental'],
+            ["gc.ps_marksweep.time",        'PS MarkSweep',         'incremental'],
+            ["gc.g1_young_generation.time", 'G1 Young Generation',  'incremental'],
+            ["gc.g1_old_generation.time",   'G1 Old Generation',    'incremental'],
+        ]},
+    'gc_ope': {
+        'options': [None, "GC Operations", "operations/s", "garbage collection", "springboot.gc_ope", "area"],
+        'lines': [
+            ["gc.copy.count",                'Copy',                 'incremental'],
+            ["gc.marksweepcompact.count",    'MarkSweepCompact',     'incremental'],
+            ["gc.parnew.count",              'ParNew',               'incremental'],
+            ["gc.concurrentmarksweep.count", 'ConcurrentMarkSweep',  'incremental'],
+            ["gc.ps_scavenge.count",         'PS Scavenge',          'incremental'],
+            ["gc.ps_marksweep.count",        'PS MarkSweep',         'incremental'],
+            ["gc.g1_young_generation.count", 'G1 Young Generation',  'incremental'],
+            ["gc.g1_old_generation.count",   'G1 Old Generation',    'incremental'],
+        ]},
+    'heap': {
+        'options': [None, "Heap Memory Usage", "KB", "heap memory", "springboot.heap", "area"],
```

Comment:
stacked?

Reply:
It should be area because:
* committed = memory alloced but not be used
* used = memory actually used

Reply:
I don't get why it can't be `stacked` :cry: 
@ktsaou